artifacts/k8s-templates/konnectivity-proxy.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: konnectivity-proxy
```

Comment:
Can we call this konnectivity-server for consistency?

Reply:
done